pkg/clouduploader/conn.go

```diff
@@ -250,3 +273,24 @@ func (c *Conn) Destroy(rw ReadWriter, opType ServerOperation) {
 		return
 	}
 }
+
+func getPartSize(config map[string]string) (val int64, err error) {
+	partSize, ok := config[MultiPartChunkSize]
+	if !ok {
+		return
```

Comment:
should we set 'err' to some value?

Reply:
or idea is to set the partsize to 0 in such cases.. it would be good to add comment if that is the case

Reply:
we are validating the value and returning error for invalid value. If value is not set then it will be 0. I didn't add comment since code is self explanatory.

Reply:
I'll add it in upcoming PR.